src/viam/sdk/robot/client.hpp

```diff
@@ -23,13 +25,62 @@ namespace viam {
 namespace sdk {
 
 using grpc::Channel;
-using viam::common::v1::ResourceName;
-using viam::common::v1::Transform;
-using viam::robot::v1::FrameSystemConfig;
 using viam::robot::v1::RobotService;
-using viam::robot::v1::Status;
+using time_point = std::chrono::time_point<long long, std::chrono::nanoseconds>;
+
+struct discovery_query {
+    discovery_query(){};
```

Comment:
Some of these new structs have constructors that set fields, others do not. is there a reason to not include the more comprehensive constructors here and below?